File: App/lib/softwares/software.dart
class Software {
  String? id;
  String name;
  String? description;
  String? area;

  Software(
      {this.id,
      required this.name,
      this.description,
      this.area});

  static Software fromMap(Map<String, dynamic> map) {
    return Software(
        id: map['id'],
        name: map['name'],
        description: map['description'],
        area: map['area']);
  }
}

File: Api/Benchmark/Contracts/Software.cs
using System.Text.Json.Serialization;

namespace Benchmark.Contracts
{
    public class Software
    {
        [JsonRequired]
        [JsonPropertyName("name")]
        public string Name { get; set; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        [JsonPropertyName("description")]
        public string Description { get; set; }

        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
        [JsonPropertyName("area")]
        public string Area { get; set; }
    }
}